dnsdb_query.py

```diff
@@ -67,22 +68,17 @@ def query_rdata_ip(self, rdata_ip, before=None, after=None):
         return self._query(path, before, after)
 
     def _query(self, path, before=None, after=None):
-        res = []
         url = '%s/lookup/%s' % (self.server, path)
 
-        params = []
+        params = {}
         if self.limit:
-            params.append('limit=%d' % self.limit)
-        if before and after:
-            params.append('time_first_after=%d' % after)
-            params.append('time_last_before=%d' % before)
-        else:
-            if before:
-                params.append('time_first_before=%d' % before)
-            if after:
-                params.append('time_last_after=%d' % after)
+            params['limit'] = self.limit
+        if before:
```

Comment:
This actually changes the functionality of --before and --after.  I'm going to incorporate your urlencode params change, though.  Thanks!